core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1237,10 +1235,10 @@ class ReplicaManager(val config: KafkaConfig,
         throw e
     }
 
-    partitionState.keys.foreach { partition =>
-      stateChangeLogger.trace(s"Completed LeaderAndIsr request correlationId $correlationId from controller $controllerId " +
-        s"epoch $controllerEpoch for the become-leader transition for partition ${partition.topicPartition}")
-    }
+    stateChangeLogger.info(s"Completed LeaderAndIsr request correlationId $correlationId from controller $controllerId " +
+      s"epoch $controllerEpoch for the become-leader transition for " +
+      s"${partitionState.size} partitions"
```

Comment:
Thinking more about it, we want per-partition granularity here. I'll add the partitions back